gtp2ogs.js

```diff
@@ -1596,13 +1614,17 @@ class Connection {
         }
 
         if (user.ranking < argv.minrank) {
-            conn_log(user.username + " ranking too low: " + user.ranking);
-            return { reject: true, msg: "Minimum rank allowed is " + argv.minrank + " (in bot ranking units), your rank is " + user.ranking + " (in bot ranking units), your rank is too low, try again when your rank is high enough." };
+            let humanReadableUserRank = rank2str(user.ranking);
+            let humanReadableMinRank = rank2str(argv.minrank);
+            conn_log(user.username + " ranking too low: " + humanReadableUserRank + " : min is " + humanReadableMinRank);
+            return { reject: true, msg: "Minimum rank is " + humanReadableMinRank + " , your rank is too low, try again when your rank is high enough." };
```

Comment:
I think you should remove the space before `: min is` and `, your`.

Reply:
the last space is fine i think
the first space maybe, but then i like spaces :laughing:

Reply:
Could you remove both spaces please? :)

Reply:
i'm not sure which to remove lol, its hard :rofl: so will leave as it is :rofl: